clang/lib/Driver/ToolChains/Arch/X86.cpp

```diff
@@ -266,13 +270,24 @@ void x86::getX86TargetFeatures(const Driver &D, const llvm::Triple &Triple,
     }
 
     bool IsNegative = Name.starts_with("no-");
+
+    bool Not64Bit = ArchType != llvm::Triple::x86_64;
```

Comment:
Does this also emit error for `-march=native`?

Reply:
It would not. We check the option here, not the feature list.